Jenkinsfile

```diff
@@ -8,9 +8,11 @@ pipeline {
     stage('Test') {
       steps {
         sh '''rm -rf build/reports
+        export LANG=en_US.UTF-8
+        export LANGUAGE=en_US.UTF-8
+        export LC_ALL=en_US.UTF-8
         eval "$(rbenv init -)"
         bundle install
-        scripts/carthage_bootstrap
```

Comment:
`carthage_bootstrap` is no longer necessary since the prebuilt dependencies are checked in